### ISSUE TITLE
Two preview window

### ISSUE THREAD

**Issue by abgaryanharutyun** [dominicstop/react-native-ios-context-menu#2]
Hey @dominicstop 

I have one question
I want to build a context menu with reaction pop-up like iMessage.

I just want to know can I build that kind of component with this package?

![eO6rt (1)](https://user-images.githubusercontent.com/5097588/97866305-24016a00-1d25-11eb-8550-f628bdd43bcc.png)

Thanks.

**Comment by dominicstop** [comment]
Hello!! sorry for the late reply, 
I did some googling and it looks like the iMessage app is using private API's to achieve this.
* When you tap on the preview, the context menu immediately closes. I've tried messing around, but currently there's no way to make the context menu preview interactive. The touch events don't pass through the preview's contents, unfortunately.

Maybe in the future, Apple will release an official API, but as of iOS 14, it's currently not possible 😔
